client/components/checkbox/style.scss

```diff
@@ -0,0 +1,75 @@
+@import '../../scss/calypso-colors';
+@import '../../scss/rem';
+
+// ==========================================================================
+// Checkboxes
+// ==========================================================================
+
+.dops-checkbox.dops-checkbox {
+	display: inline-block;
+	box-sizing: border-box;
+	margin: 2px 0 0;
+	padding: 7px 14px;
+	width: 16px;
```

Comment:
@MichaelArestad Do we have that rem function in dops-comp? If so, should we be using it here?